src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java

```diff
@@ -89,14 +89,18 @@ public static Object[] isIcebergMode() {
   private SnowflakeStreamingIngestClientInternal<StubChunkData> client;
   private MockSnowflakeServiceClient.ApiOverride apiOverride;
 
+  private Properties prop;
```

Comment:
does it really need to be a class member and be part of "test state" ? Add a static method that returns a new Properties object on every call (i see you need props in a testcase down below)

Reply:
or probably just add a TestUtils.createProps(isIcebergMode) static method and use it everywhere

Reply:
Created `createProps`.